src/reference/asciidoc/amqp.adoc

```diff
@@ -644,6 +644,17 @@ The `cacheMode` property (`CHANNEL` or `CONNECTION` is also included).
 .JVisualVM Example
 image::images/cacheStats.png[align="center"]
 
+[[auto-recovery]]
+===== RabbitMQ Automatic Connection/Topology recovery
+
+Since the first version of Spring AMQP, the framework has provided its own connection and channel recovery in the event of a broker failure.
+Also, as discussed in <<broker-configuration>>, the `RabbitAdmin` will re-declare any infrastructure beans (queues etc) when the connection is re-established.
+It therefore does not rely on the https://www.rabbitmq.com/api-guide.html#recovery[Auto Recovery] that is now provided by the `amqp-client` library.
+Spring AMQP now uses the `4.0.x` version of `amqp-client`, which has auto recovery enabled by default.
```

Comment:
If we assume to still support pre-4.0 clients, this sentence should be adjusted a bit:

Spring AMQP now **supports** the `4.0.x`...